languages.toml

```diff
@@ -3989,3 +3989,16 @@ indent = { tab-width = 4, unit = "    " }
 [[grammar]]
 name = "vento"
 source = { git = "https://github.com/ventojs/tree-sitter-vento", rev = "3321077d7446c1b3b017c294fd56ce028ed817fe" }
+
+[[language]]
+name = "nginx"
+scope = "source.nginx"
+injection-regex = "nginx"
+file-types = ["conf"]
```

Comment:
like in https://github.com/helix-editor/helix/pull/12156 we should use globs for things like `nginx.conf` and `*.conf` files within some nginx directory names (`nginx/conf.d/*.conf`? it's been a while since I used nginx)

Reply:
I've added some common glob patterns